src/assemble_workflow/dist.py

```diff
@@ -116,10 +118,19 @@ def __build__(self, name: str, dest: str) -> None:
                     zip.write(fn, fn[noleadingdirlen:])
 
 
+class DistDeb(Dist):
+
+    def __extract__(self, dest: str) -> None:
+        self.bundle_linux_distro.extract('deb', dest)
```

Comment:
This class should create an instance of `BundleLinuxDistroDeb` in its init.

Reply:
Removed `BundleLinuxDistro` and using `BundleLinuxDeb` here instead.

Should we rename `BundleLinuxDeb` to `BundleLinuxDistroDeb` (and same for rpm)?

Reply:
I checked with Peter and he agreed with the current solution.